From 8fbf1e7d0f071a65ec2a18a6b3cca35ef7a6c21d Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Tue, 20 Jul 2021 17:59:40 +0800
Subject: [PATCH] [riscv64] Add baseline_riscv64_inl.h into BUILD.gn

Change-Id: I82464ad3ae81843b8e3ac940c079218a55aad67e
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3041145
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Commit-Queue: Hannes Payer <hpayer@chromium.org>
Reviewed-by: Ji Qiu <qiuji@iscas.ac.cn>
Reviewed-by: Hannes Payer <hpayer@chromium.org>
Cr-Commit-Position: refs/heads/master@{#76068}
---
 BUILD.gn | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/BUILD.gn b/BUILD.gn
index a692bcba621..3e5b71d01ef 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -3483,6 +3483,8 @@ v8_header_set("v8_internal_headers") {
     ]
   } else if (v8_current_cpu == "riscv64") {
     sources += [  ### gcmole(arch:riscv64) ###
+      "src/baseline/riscv64/baseline-assembler-riscv64-inl.h",
+      "src/baseline/riscv64/baseline-compiler-riscv64-inl.h",
       "src/codegen/riscv64/assembler-riscv64-inl.h",
       "src/codegen/riscv64/assembler-riscv64.h",
       "src/codegen/riscv64/constants-riscv64.h",
-- 
2.35.1

